tests/responses/test_fileresponse.py

```diff
@@ -80,3 +80,33 @@ def test_unicode_attachment(self):
             response['Content-Disposition'],
             "attachment; filename*=utf-8''%E7%A5%9D%E6%82%A8%E5%B9%B3%E5%AE%89.odt"
         )
+
+    def test_file_to_stream_closes_response(self):
+        # Closing the file_to_stream should call FileResponse.close(), even
```

Comment:
We usually avoid _should_ verbiage in tests.

e.g.

```
# Closing file_to_stream calls FileResponse.close() even when
# file-like object doesn't have a close() method.

Reply:
> We usually avoid should verbiage in tests.

You might want to add this to / clarify this in the [coding style docs](https://docs.djangoproject.com/en/2.2/internals/contributing/writing-code/coding-style/#python-style). It currently starts with this--

> In test docstrings, state the expected behavior that each test demonstrates. Don’t include preambles such as “Tests that” or “Ensures that”.